Libraries/Image/RCTAnimatedImage.m

```diff
@@ -87,6 +87,10 @@ - (NSUInteger)imageLoopCountWithSource:(CGImageSourceRef)source
     NSNumber *gifLoopCount = gifProperties[(__bridge NSString *)kCGImagePropertyGIFLoopCount];
     if (gifLoopCount != nil) {
       loopCount = gifLoopCount.unsignedIntegerValue;
+      // A loop count of 1 means it should repeat twice, 2 means, thrice, etc.
+      if (loopCount != 0) {
```

Comment:
Hey @zhongwuzw ,

I think we could simplify this, we could assign loopCount = 0 and then when returning from the function we could return loopCount + 1. This way we get rid of this condition. What do you think? 

Thanks!

Reply:
@sammy-SC Emm, if we return `loopCount + 1`, the min value is 1? but we need to return 0 if `gifLoopCount` exist and its value is 0.

Reply:
Oh, I see :) 

Thanks for fixing this.